routes/api.php

```diff
@@ -1,4 +1,4 @@
-<?php
+<? php
```

Comment:
Not sure about that. :D 

![image](https://media0.giphy.com/media/GVtmJrnQj92qvjcyiq/giphy.gif?cid=ecf05e47e2vybm3eyb6quozn45ewgjg5t74tspqj3mtzldyr&rid=giphy.gif&ct=g)

Reply:
oops, i think this is my formatter's fault